clipper_admin/clipper_admin/deployers/pytorch.py

```diff
@@ -0,0 +1,188 @@
+        The PyTorch model to save.
+    default_output : str, optional
+        The default output for the application. The default output will be returned whenever
+        an application is unable to receive a response from a model within the specified
+        query latency SLO (service level objective). The reason the default output was returned
+        is always provided as part of the prediction response object. Defaults to "None".
+    version : str, optional
+        The version to assign this model. Versions must be unique on a per-model
+        basis, but may be re-used across different models.
+    slo_micros : int, optional
+        The query latency objective for the application in microseconds.
+        This is the processing latency between Clipper receiving a request
+        and sending a response. It does not account for network latencies
+        before a request is received or after a response is sent.
+        If Clipper cannot process a query within the latency objective,
+        the default output is returned. Therefore, it is recommended that
+        the SLO not be set aggressively low unless absolutely necessary.
+        100000 (100ms) is a good starting value, but the optimal latency objective
+        will vary depending on the application.
+    labels : list(str), optional
+        A list of strings annotating the model. These are ignored by Clipper
+        and used purely for user annotations.
+    registry : str, optional
+        The Docker container registry to push the freshly built model to. Note
+        that if you are running Clipper on Kubernetes, this registry must be accesible
+        to the Kubernetes cluster in order to fetch the container from the registry.
+    base_image : str, optional
+        The base Docker image to build the new model image from. This
+        image should contain all code necessary to run a Clipper model
+        container RPC client.
+    num_replicas : int, optional
+        The number of replicas of the model to create. The number of replicas
+        for a model can be changed at any time with
+        :py:meth:`clipper.ClipperConnection.set_num_replicas`.
+    """
+
+    clipper_conn.register_application(name, input_type, default_output,
+                                      slo_micros)
+    deploy_pytorch_model(clipper_conn, name, version, input_type, func,
+                         pytorch_model, base_image, labels, registry,
+                         num_replicas)
+
+    clipper_conn.link_model_to_app(name, name)
+
+
+def deploy_pytorch_model(
+        clipper_conn,
+        name,
+        version,
+        input_type,
+        func,
+        pytorch_model,
+        base_image="clipper/pytorch-container:{}".format(__version__),
+        labels=None,
+        registry=None,
+        num_replicas=1):
+    """Deploy a Python function with a PySpark model.
+    The function must take 3 arguments (in order): a SparkSession, the PySpark model, and a list of
```

Comment:
Update this description. The function no longer takes a `SparkSession` object. Replace all instances of `PySpark` of `PyTorch`